src/OrchardCore.Modules/OrchardCore.Security/Views/SecurityHeadersSettings.Edit.cshtml

```diff
@@ -17,87 +198,73 @@
         </option>
     </select>
     <span asp-validation-for="FrameOptions"></span>
-    <span class="hint">@T["Tells the browser whether you want to allow your site to be framed or not."]</span>
+    <span class="hint">@T["Tells the browser whether you want to allow your site to be displayed in a frame or not."]</span>
 </div>
-<div class="form-group" asp-validation-class-for="PermissionsPolicy">
+<div class="form-group">
     <label asp-for="PermissionsPolicy">@T["Permissions Policy"]</label>
```

Comment:
https://user-images.githubusercontent.com/1976647/167738711-a95112ed-4a89-4ed6-988e-e575b5c5358c.mp4

Reply:
This is odd, I'm sure this happen after the latest changes with `ConfigureSecuritySettings()`, let me check

Reply:
Seems the issue here we need a way to check if the settings comes from database or appsettings, I'm thinking to compare the `SecurityHeadersOptions` and `IOptionsSnapshot<SecurityHeadersOptions>` value, the only limitation I see if the settings equivalent to the defaults I can't know if I will use the database value or the appsettings one

I'm not how you address this issue in the Lombiq repo?

Reply:
FYI the bug introduced in the above video fixed in https://github.com/OrchardCMS/OrchardCore/pull/11538/commits/a712dab4935bc49f6d2c0fc5c4bee2e8b5797569, but again introducing the ConfigureSecuritySettings() introduced another issue which mentioned in the above thread

Reply:
Please search for how `DemoSettings` is used in the Training Demo, it demonstrates everything.

Reply:
If you are refering to https://github.com/Lombiq/Orchard-Training-Demo-Module it was using a normal settings

Reply:
`DemoSettings` is both site settings and appsettings, the exact thing needed here.

Reply:
This is confusing a little bit, in our case the use might set `XFrame-Options` to self and its none on the settings screen. If we need to go with this way, no need to use snapshot options in the driver, they should be applied when the `SecurityHeadersOptions` is configured in `Startup`

Reply:
I just mean that that's the approach we've taken in the Training Demo. Feel free to do it in a way that the settings screen also displays the static config.